ISSUE_TEMPLATE

```diff
@@ -0,0 +1,40 @@
+Before filing an issue:
+-----------------------
+- Attempt to reproduce the issue using 
+  [this sample application](https://github.com/forcedotcom/ServiceSDK-Android/)
```

Comment:
Great idea! Since this links to the entire repo, you might consider specifying exactly which sample application (i.e. "SnapinsSDKExample"), or linked to the specific example (i.e. https://github.com/forcedotcom/ServiceSDK-Android/tree/master/Examples/SnapinsSDKExample).

Reply:
Updated to `Examples/SnapinsSDKExample`.